samples/security/spire/README.md

```diff
@@ -0,0 +1,44 @@
+# Integrating SPIRE through Envoy SDS
+
+There are currently multiple forms of integrating third-party Certificate Authorities (CA) whitin Istio. This example deploys a setup of [SPIRE](https://github.com/spiffe/spire) (the SPIFFE Runtime Environment) as an Exernal CA, through [Envoy's SDS API integration](https://github.com/alexandrealvino/istio.io/blob/6eff1cd45dfce244978cafc10a88f7620c0438fc/content/en/docs/ops/integrations/spire/index.md).
```

Comment:
IMO the phrase `There are currently multiple forms of integrating third-party Certificate Authorities (CA) whitin Istio.` is irrelevant in this context, we are not explaining in this README which are those multiple forms, but specifically how to integrate with SPIRE.

Reply:
Thanks @sanderson042, applied all the minor formatting advices, and also summarized the demo overview in the next paragraph.

Good point @maxlambrecht , It's arguable that because it's very short (<1 line) and it gives a contextual intro to the example, it could be valuable... but I guess it's more of a left over from when we were over-explaining a lot... Anyways, I'll remove it 😆